src/test/kotlin/com/example/serverFactoryAPI/controllers/UserControllerTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.serverFactoryAPI.controllers
```

Comment:
@MarkoLazovicCode Tests should not be contained in same directory source code structure as application code. See Gradle guidelines for directories for tests. See:
https://docs.gradle.org/current/userguide/organizing_gradle_projects.html